nbs/examples/Robust_Regression.ipynb

```diff
@@ -0,0 +1,786 @@
+{
```

Comment:
<p>Remove this table print, occupies a lot of space, is not interesting.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Nixtla/neuralforecast/pull/582/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>